pages/Cards.md

```diff
@@ -48,6 +48,6 @@ collapsed:: true
 - [[Resources]]
 	- [Augmenting Long-term Memory](http://augmentingcognition.com/ltm.html)
 	- [SM5](https://www.supermemo.com/en/archives1990-2015/english/ol/sm5) by supermemo
-- TODO Organize this page. Maybe split out tutorial. Add Tienson's latest functionality #docs
+- TODO 
```

Comment:
Prefer to have `#docs` on the task itself as all the child blocks are things that also need to be done and would be useful to see in referenced contexts. I'll add a note about this in the contributing section